board.py

```diff
@@ -215,11 +215,26 @@ def insertCard(self, inputArgs):
         if not self.cardLocationIsValidSpot(positionNewCard, positionSecondTile, newCard):
             print("The location where you want to place your card is not valid.")
             return None
-        self.board[positionNewCard[1]][positionNewCard[0]] = newCard.activeSide.tile1
-        self.board[positionSecondTile[1]][positionSecondTile[0]] = newCard.activeSide.tile2
-
-        self.nbrCards += 1
-
+        
+         if int(inputArgs[3]) == 1 and (inputRotCode == 2 or inputRotCode == 4 or inputRotCode == 6 or inputRotCode == 8):
+            self.nbrCards += 1
+            if inputRotCode == 2:
+                self.board[positionNewCard[1] + 1][positionNewCard[0]] = newCard.activeSide.tile1
+                self.board[positionSecondTile[1] - 11][positionSecondTile[0]] = newCard.activeSide.tile2
+            elif inputRotCode == 4:
+                self.board[positionNewCard[1]][positionNewCard[0]] = newCard.activeSide.tile1
+                self.board[positionSecondTile[1] - 12][positionSecondTile[0]] = newCard.activeSide.tile2
+            elif inputRotCode == 6:
+                self.board[positionNewCard[1]][positionNewCard[0]] = newCard.activeSide.tile2
+                self.board[positionSecondTile[1]-10][positionSecondTile[0]] = newCard.activeSide.tile1
+            elif inputRotCode == 8:
```

Comment:
please change this to
else: # if inputRotCode == 8:
(if inputRotCode is not 2, 4, 8, then it has to be 8)

Reply:
The rotation code = 8 command should stay as an elif because it should only be executed if the row is 1 and the rotation code is 8. There is a possibility that the row is 1 and rotation code is 5, in which case we go to the else condition. Changing the inputRotCode == 8 to an else instead of an elif will enforce its execution even in the cases where we don't want it.

Reply:
It cannot be 1, 3, 5 or 7 since that elif is within "and (inputRotCode == 2 or inputRotCode == 4 or inputRotCode == 6 or inputRotCode == 8)"